_posts/2019-10-24-socratic-seminar-97.md

```diff
@@ -0,0 +1,105 @@
+- [600,000th block](https://twitter.com/pwuille/status/1185346625897517056)
+- [Stale block](https://twitter.com/bitmexresearch/status/1184408265360334849?s=21)
+
+## CVEs and Research
+
+#### Research
+
+- [ABC: Asynchronous Blockchain without Consensus](http://arxiv.org/abs/1909.10926)
+- [Boomerang: Redundancy Improves Latency and Throughput in Payment Networks](http://arxiv.org/abs/1910.01834)
+- [Human readable raw transactions](https://twitter.com/notgrubles/status/1180861313452511233)
+- [Hijacking Routes in Payment Channel Networks: A Predictability Tradeoff](http://arxiv.org/abs/1909.06890)
+- [Privacy-Utility Tradeoffs in Routing Cryptocurrency over Payment Channel Networks](http://arxiv.org/abs/1909.02717)
+- [Short Paper: Towards Characterizing Sybil Attacks in Cryptocurrency Mixers](https://eprint.iacr.org/2019/1111)
+- [Simplicity's first transaction](https://lists.ozlabs.org/pipermail/simplicity/2019/000018.html)
+- [Transparent SNARKs from DARK Compilers](https://eprint.iacr.org/2019/1229.pdf)
+- [Binary quadtratic forms](http://diyhpl.us/%7Ebryan/papers2/bitcoin/vdf-competition-classgroups-binary-quadratic-form.pdf)
+- [Bitcoin Chain Width ExpansionDenial-of-Service Attacks](https://bcoin.io/papers/bitcoin-chain-expansion.pdf)
+
+
+#### InfoSec
+
+- [lnd, c-lightning, eclair CVE-2019-12998 / CVE-2019-12999 /	CVE-2019-13000](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-September/002174.html) [Rusty broadcast](https://twitter.com/i/broadcasts/1yqkdnryandjv)
+- [MakerDAO - Steal ALL collateral during liquidation by exploiting lack of validation in `flip.kick`](https://hackerone.com/reports/684092)
+- [Android: Use-After-Free in Binder driver](https://bugs.chromium.org/p/project-zero/issues/detail?id=1942#c7)
+- [ipwndfu - open-source jailbreaking tool for many iOS devices](https://github.com/axi0mX/ipwndfu)
+- [Sudo Flaw](https://thehackernews.com/2019/10/linux-sudo-run-as-root-flaw.html)
+- [Broken Metre: Attacking Resource Metering in EVM](http://arxiv.org/abs/1909.07220)
+- [China’s Maxim – Leave No Access PointUnexploited: The Hidden Story of China Telecom’sBGP Hijacking](https://scholarcommons.usf.edu/cgi/viewcontent.cgi?article=1050&context=mca)
+- [Tracking the PlusToken Whale: Attempted Bitcoin Laundering and Its Impact on Wasabi Wallet](https://medium.com/@ErgoBTC/tracking-the-plustoken-whale-attempted-bitcoin-laundering-and-its-impact-on-wasabi-wallet-787c0d240192)
+
+## Pull Requests and repo updates
+
+#### Bitcoin Core
+
+#### BIPs
+
+- [Transaction announcements reconciliation](https://github.com/naumenkogs/bips/blob/bip-reconcil/bip-reconcil.mediawiki)
+- [Taproot Review](https://github.com/ajtowns/taproot-review)
+
+#### lnd
+
+- [0.8.0-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.8.0-beta)
+
+#### c-lightning
+
+#### BOLTs
+
+- Rusty Russell BOLT Broadcasts:
+  - [BOLT 4](https://twitter.com/i/broadcasts/1drkzlvmoldjb)
```

Comment:
all of these links are broken except **BOLT 7 part 1**

Reply:
Fixing, thanks.